storage/remote/client.go

```diff
@@ -50,7 +61,7 @@ var (
 			Name:      "read_queries_total",
 			Help:      "The total number of remote read queries.",
 		},
-		[]string{remoteName, endpoint, "code"},
+		[]string{remoteName, endpoint, "response_type", "code"},
 	)
 	remoteReadQueries = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
```

Comment:
Should we change this to also provide info if sampled/chunked?

Reply:
I wanted to, but I don't think there's a way to know if it's sampled vs. chunked until we get a response from the server.